.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] Please add the gitignore file also [see here](https://user-images.githubusercontent.com/61562626/128883503-07f8d8f7-7227-4f50-9407-0b780db78904.jpeg)

Reply:
Is this the only thing I have to fix then I will be good to proceed?
I have only one review left?